tests/api-compatibility/acceptable-breakages-vReference.txt

```diff
@@ -1271,4 +1271,3794 @@ CannotChangeAttribute : Attribute 'Android.Runtime.RegisterAttribute' on 'Androi
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.PasswordComplexity Android.App.Admin.DevicePolicyManager.PasswordComplexityLow' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.PasswordComplexity Android.App.Admin.DevicePolicyManager.PasswordComplexityMedium' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.PasswordComplexity Android.App.Admin.DevicePolicyManager.PasswordComplexityNone' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.PrivateDnsMode Android.App.Admin.DevicePolicyManager.PrivateDnsModeOff' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.PrivateDnsMode Android.App.Admin.DevicePolicyManager.PrivateDnsModeOpportunistic' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.PrivateDnsMode Android.App.Admin.DevicePolicyManager.PrivateDnsModeProviderHostname' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.PrivateDnsMode Android.App.Admin.DevicePolicyManager.PrivateDnsModeUnknown' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.PrivateDnsSet Android.App.Admin.DevicePolicyManager.PrivateDnsSetErrorFailureSetting' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.PrivateDnsSet Android.App.Admin.DevicePolicyManager.PrivateDnsSetErrorHostNotServing' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.PrivateDnsSet Android.App.Admin.DevicePolicyManager.PrivateDnsSetNoError' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.ResetPasswordFlags Android.App.Admin.DevicePolicyManager.ResetPasswordDoNotAskCredentialsOnBoot' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.ResetPasswordFlags Android.App.Admin.DevicePolicyManager.ResetPasswordRequireEntry' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.UserManagementFlags Android.App.Admin.DevicePolicyManager.LeaveAllSystemAppsEnabled' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Android.App.Admin.UserManageme
```

Comment:
This and the following messages don't make sense to me.  These are *not* the `[Obsolete]` attribute changes above; instead, it sounds like we're no longer emitting some of these methods?  I am confused. :-(

Reply:
...oh; these are the "`protected` members within `sealed` classes" category of changes.  `PreferenceScreen` is `sealed`, but `PreferenceScreen.OnCreateRootAdapter()` is `protected`, which makes no sense.

As such, they *do* make sense.